internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -642,6 +642,12 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"run_command_enabled": {
```

Comment:
A bit weird calling this parameter enabled as the Azure SDK parameter is called disabled. Any opinion on this?

Reply:
`something_enabled` is consistent with the provider, so this is fine 👍🏼